pipelines/build/common/config_regeneration.groovy

```diff
@@ -285,7 +285,19 @@ class Regeneration implements Serializable {
                             context.println "[INFO] Checking if ${pipeline} is running..." //i.e. openjdk8-pipeline
 
                             def pipelineInProgress = queryAPI("${jenkinsBuildRoot}/job/${pipeline}/lastBuild/api/json?pretty=true&depth1")
-                            inProgress = pipelineInProgress.building as Boolean
+
+                            // If query fails, check to see if the pipeline been run before
+                            if (pipelineInProgress == null) {
+                                def getPipelineBuilds = queryAPI("${jenkinsBuildRoot}/job/${pipeline}/api/json?pretty=true&depth1")
+
+                                if (getPipelineBuilds.builds == []) {
```

Comment:
Can you get null instead of [] returned?

Reply:
I checked and it seems to always be an empty array by default. I also ran the PR tester with this code when the `builds` field was empty and it worked fine